### ISSUE TITLE
cpp:function could use inline markup

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#680]
It would be useful for cpp:function to support inline markup constructs:

```
.. cpp:function:: double log(double x)

   :param x: a positive number
   :returns: the natural logarithm of `x`
   :throws: std::domain_error
```

Note the substitution of `throws` for `raises`, since C++ uses a different keyword. Also note the absence of the type-oriented constructs, since those are explicit in C++ function signatures already. 

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/680
- Originally reported by: Anonymous
- Originally created at: 2011-04-17T07:10:33.971

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2011-09-22 08:36:54+00:00_

Closes #680: allow doc fields in cpp domain directives.

→ <<cset e5e44202b659c0def17bdb1720a4cae2eee07327>>

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-10-18 07:32:18+00:00_

Removing version: 1.0.7 (automated comment)
